toponetx/classes/simplicial_complex.py

```diff
@@ -591,7 +578,19 @@ def get_boundaries(simplices, min_dim=None, max_dim=None):
         return face_set
 
     def remove_maximal_simplex(self, simplex):
-        """Remove maximal simplex from simplicial complex."""
+        """Remove maximal simplex from simplicial complex.
+
+        Note
+        -----
```

Comment:
@ninamiolane For whatever reason, pydocstyle only checks D409 (the one that checks whether the underline's length matches the section name) for _known_ section names, i.e., those section names that are defined in NumPy's style guide. See [this call](https://github.com/PyCQA/pydocstyle/blob/07f6707e2c5612960347f7c00125620457f490a7/src/pydocstyle/checker.py#L788) where D409 is eventually triggered in a sub-function but only `NUMPY_SECTION_NAMES` are checked.

Since in this case the section name is wrong (it should be `Notes` as per the style guide), it is just ignored. There is no way to configure pydocstyle to check all section names as far as I can see.